src/misc/AttachShadowPolyfill.ts

```diff
@@ -0,0 +1,43 @@
+import { $$ } from '../utils/Dom';
+import 'styling/_AttachShadowPolyfill';
+
+export async function attachShadow(element: HTMLElement, init: ShadowRootInit): Promise<ShadowRoot | HTMLElement> {
+  if (element.attachShadow) {
+    return element.attachShadow(init);
+  }
```

Comment:
On any browser other than IE11, `attachShadow` will be used instead of an iframe.

Reply:
What are the benefits of using two different rendering approaches? Could it be simpler to just use an iframe since it is supported in all browsers? I think that would remove shadow checks in this file and the next.

Reply:
Good point, I'll make it always use iframes.